src/include/stir/DynamicDiscretisedDensity.h

```diff
@@ -171,6 +171,9 @@ class DynamicDiscretisedDensity: public ExamData
   void set_time_frame_definitions(const TimeFrameDefinitions& time_frame_definitions) 
   {this->exam_info_sptr->set_time_frame_definitions(time_frame_definitions);}
 
+  void set_scanner(const shared_ptr<Scanner> scanner_sptr)
```

Comment:
lines 167 and 169 I cannot understand the `const`

Reply:
Lines 167 and 169 are not methods that I have altered.